### PR TITLE
budget: initial commit

### PR DESCRIPTION
**Pull request by philips**
Initial commit on how to request funding from the steering committee.
The issue templates will move to github.com/kubernetes/funding

### CODE REVIEW

**Comment by spiffxp** [comment]
/hold
for comment by other steering members

/lgtm

**Comment by dims** [comment]
Thanks for doing this @philips +1 from me.

**Comment by philips** [comment]
Added to the agenda for the June 4th steering committee meeting

**Comment by philips** [comment]
The issue template I will file on kubernetes/org

```
### New Repo, Staging Repo, or migrate existing

New repo

### Requested name for new repository

funding

### Which Organization should it reside

kubernetes

### If not a staging repo, who should have admin access

@kubernetes/steering-committee 

### If not a staging repo, who should have write access

@kubernetes/steering-committee 

### If not a staging repo, who should be listed as approvers in OWNERS

@kubernetes/steering-committee 

### If not a staging repo, who should be listed in SECURITY_CONTACTS

@kubernetes/steering-committee 

### What should the repo description be

file an issue to request funding for Kubernetes needs

### What SIG and subproject does this fall under in sigs.yaml

Steering Committee

### Approvals

https://github.com/kubernetes/steering/issues/46
https://github.com/kubernetes/steering/pull/106

### Additional context for request

None.
```

File: budget.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+title: request: one-time $NNNUSD to fund MMM
+about: a non-recurring request with easiest process
+---
+
+<!--
+Please fill out the items and provide supporting evidence for request
+-->
+
+- **Description**: 
+- **Requesting SIG**:
```
Comment (nikhita) [inline budget.md]: Drive-by question: would it be helpful to have a field for  SIG Approval here as well (just like [requiring SIG leads to sign off on creating new repos](https://github.com/kubernetes/org/blob/master/.github/ISSUE_TEMPLATE/repo-create.md#approvals)) or adding a note that the request would require approval from the relevant SIG chair?

However that probably means there is very little difference between `one-time-200.md` and `one-time-200-plus.md`...

File: budget.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- **Cost**:
+
+-.github/ISSUE_TEMPLATE/one-time-200-plus.md
+
+--
+name: One-time funding under $200USD
```
Comment (zegl) [inline budget.md]: I'm guessing that this should be the document for _over_ $200.

```suggestion
name: One-time funding over $200USD
```

Comment (philips) [reply budget.md]: oops, thanks, will fix when we move over to the repo

**Comment by philips** [comment]
thanks for the LGTMs, request for a new repo here: https://github.com/kubernetes/org/issues/899

**Comment by spiffxp** [comment]
https://github.com/kubernetes/funding has been created, please update as needed

**Comment by spiffxp** [comment]
Looking for LGTM on https://github.com/kubernetes/community/pull/3789

**Comment by philips** [comment]
Closing. Moving stuff to github.com/kubernetes/funding

File: budget.md (hunk 1 above)
Comment (philips) [reply budget.md]: good idea @nikhita will add to README.

**Pull request closed without merging**